notes.py

```diff
@@ -144,16 +157,23 @@ class ChordName:
         'm7': [0, 3, 7, 10],
         'm7b5': [0, 3, 6, 10],
         'dim7': [0, 3, 6, 9],
-        'aug7': [0, 4, 8, 11],
+        'aug7': [0, 4, 8, 10],
+    }
+    FLAT_KEYS = ['C', 'F', 'Bb', 'Eb', 'Ab', 'Db', 'Gb', 'Cb', 'Fb', 'Bbb', 'Ebb', 'Abb', 'Dbb']
+    SHARP_KEYS = ['G', 'D', 'A', 'E', 'B', 'F#', 'C#', 'G#', 'D#', 'A#', 'E#', 'B#', 'F##']
+    KEY_BIAS = {
+        **{k: 'b' for k in FLAT_KEYS},
+        **{k: '#' for k in SHARP_KEYS},
     }
 
     def __init__(self, chord_name: str):
-        chord_note = None
+        chord_notes = []
         for note_name in Note.ALL_NOTES_NAMES:
             if chord_name.startswith(note_name):
-                chord_note = note_name
-                break
-        if chord_note is None:
+                chord_notes.append(note_name)
+        try:
+            chord_note = max(chord_notes, key=len)
```

Comment:
This was previously a bug, where, say, for the key of `F#`, the loop would break after hitting `startswith('F')`